extensions/interactions/AlgebraicExpressionInput/directives/algebraic-expression-input-rules.service.ts

```diff
@@ -55,6 +55,20 @@ export class AlgebraicExpressionInputRulesService {
     return answer === inputs.x;
   }
 
+  MatchesUpToTrivialManipulations(
+      answer: AlgebraicExpressionAnswer,
+      inputs: AlgebraicExpressionRuleInputsWithoutPlaceholder
+  ): boolean {
+    let mis = new MathInteractionsService();
+    let numericExpressionRuleService = (
+      new NumericExpressionInputRulesService());
```

Comment:
Why do we init these services here? Shouldn't this be placed in some kind of constructor?

Reply:
Done.